CHANGELOG.txt

```diff
@@ -1,6 +1,10 @@
 CHANGELOG
 =========
 
+v3.2.1
+------
+* Change version in package.json, package-lock.json and cartridges/int_alma/package.json
```

Comment:
```suggestion
* Change version
```

The changelog bullets must be concise and absolutely not talk about code (like a commit message)